src/common/SPELLS/paladin.ts

```diff
@@ -123,6 +123,11 @@ const spells = spellIndexableList({
     icon: 'spell_holy_flashheal',
     manaCost: 2200,
   },
+  HOLY_SHOCK_CAST: {
+    id: 20473,
+    name: 'Holy Shock',
+    icon: 'spell_holy_searinglight',
+  },
```

Comment:
This is a talent (tho perma selected). Warcraft logs sends over Infusion of Light events (buff gained probably?) with this spell ID which leads to <SpellLink> showing the wrong icon/spell name. Seems like the easiest fix for now, even though it duplicates from the TALENTS configuration.

Long term I suspect <SpellLink> (and related) should look up spell ids in TALENTS if it doesn't find anything in SPELLS. Would be happy to attempt if that sounds like a good change.

Reply:
on live, all these Infusion of Light should be Holy Shock instead
![image](https://user-images.githubusercontent.com/229650/211173641-a0a2153e-f8c1-44ae-adbc-f7363132ea58.png)

Reply:
solved with help from discord by passing spell objects instead of ids.